src/components/PersonModal/PersonModal.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types/Person';
+
+interface Props {
+  person: Person;
+  selectedPersonSlug: string;
+  people: Person[];
+}
+
+export const PersonModal: React.FC<Props> = ({
+  person,
+  selectedPersonSlug,
+  people,
+}) => {
+  const {
+    slug, sex, name, died, born, motherName, fatherName,
+  } = person;
+
+  let motherLink = null;
+
+  if (motherName) {
+    const motherPerson = people.find((p) => p.name === motherName);
+
+    if (motherPerson) {
+      motherLink = (
+        <Link
+          to={`/people/${motherPerson.slug}`}
+          className="has-text-danger"
+        >
+          {motherName}
+        </Link>
+      );
+    } else {
+      motherLink = motherName;
+    }
+  } else {
+    motherLink = '-';
+  }
+
+  let fatherLink = null;
+
+  if (fatherName) {
+    const fatherPerson = people.find((p) => p.name === fatherName);
+
+    if (fatherPerson) {
+      fatherLink = (
+        <Link to={`/people/${fatherPerson.slug}`}>{fatherName}</Link>
+      );
+    } else {
+      fatherLink = fatherName;
+    }
+  } else {
+    fatherLink = '-';
+  }
```

Comment:
move it all into component `<FatherLink />` and use early return pattern as well